src/engine.rs

```diff
@@ -630,9 +630,6 @@ impl Reedline {
 
         let result = self.read_line_helper(prompt);
 
-        #[cfg(not(target_os = "windows"))]
```

Comment:
I kind of wonder if this was originally just a typo and should've really been `#[cfg(target_os = "windows"))]` because bracketed paste doesn't work in windows since crossterm doesn't support it yet.

Reply:
The author of the original issue was using Linux. Though he was using the Windows Terminal (using debian via WSL2).

Thus it might be a Windows Terminal issue? I don't think anyone tried it to reproduce it with Windows Terminal specifically. And it seems nobody was actually able to reproduce it accept #theAkito himself.

Reply:
I can try to reproduce with [WezTerm](https://wezfurlong.org/wezterm/) on Windows. Just tell me which commits to try & any additional useful information...